glide.yaml

```diff
@@ -1,30 +1,26 @@
 package: github.com/target/pod-reaper
 import:
 - package: k8s.io/apimachinery
-  version: c1b2a29e2e9d121b9b7d818cb0d84d8098783236
+  version: =v0.17.0
   subpackages:
-    - /pkg/labels
-    - /pkg/selection
+  - pkg/labels
+  - pkg/selection
 - package: k8s.io/client-go
-  version: =v2.0.0
+  version: =v0.17.0
```

Comment:
Am I reading this right that we are going _backwards_ in version number here?

Reply:
I know it works, but that's a REALLY good question.

Reply:
Looking at the `client-go` versions, it looks like they switched mechanisms to match k8s versions, so `0.17.0` maps to `1.17.0`.  So it does appear this is a newer version, though not obviously so.

Reply:
I didn't have a chance to look before you, but I was *hoping* it'd be something like that... Doing to cause a mess if k8s ever get to 2.0.0!